Pilot/pilotTools.py

```diff
@@ -214,53 +214,97 @@ def listdir(directory):
     return contents
 
 
-def getFlavour(ceName):
+def getSubmitterInfo(ceName):
```

Comment:
Out if being conservative for the Pilot, please do not change this name.

Reply:
If you want to be conservative, just keeping the same name if not enough as I am also changing the returned result...

I guess I would actually need to define 2 different methods: `getFlavour()`, which should be marked as deprecated and `getSubmitterInfo()`. What do you think?

Reply:
In my latest commit, I define both methods, `getFlavour()` is declared deprecated. I used `warnings`, which should be compatible from python2.6 to python3.11.
Any opinion?

Reply:
Perfect.